src/fiber.cr

```diff
@@ -65,9 +63,13 @@ class Fiber
     @@fibers.delete(fiber)
   end
 
+  # :nodoc:
+  def self.stack_pool
+    @@stack_pool ||= StackPool.new
```

Comment:
Does this need to be lazily created? It will never be `nil` in any program.

Reply:
Thanks, I overlooked that!